projects/python.org/package.yml

```diff
@@ -45,73 +45,93 @@ build:
+
+        # 3.7 aborts on linux+aarch64 with this
+        ARGS=${ARGS/--with-ensurepip/--without-ensurepip}
+      fi
 
-    ./configure $ARGS
-    make --jobs {{ hw.concurrency }}
-    make install
+    - ./configure $ARGS
+    - make --jobs {{ hw.concurrency }}
+    - make install
 
     # customize `pip install` locations to tea defaults
-    cp props/sitecustomize.py {{prefix}}/lib/python{{version.marketing}}
+    - cp props/sitecustomize.py {{prefix}}/lib/python{{version.marketing}}
 
     # provide unversioned symlinks
-    cd {{prefix}}/bin
-    for x in python idle pydoc; do
-      ln -sf ${x}{{ version.marketing }} $x
-    done
-    ln -sf python{{ version.marketing }}-config python-config
+    - run: |
+        for x in python idle pydoc; do
+          ln -sf ${x}{{ version.marketing }} $x
+        done
+        ln -sf python{{ version.marketing }}-config python-config
+      working-directory: ${{prefix}}/bin
 
     # we provide the binaries in a separate package
     # but we don’t use --without-ensurepip because stuff expects python to
     # come bundled with setuptools
-    find . -mindepth 1 -maxdepth 1 -type f -name 'pip*' -exec rm {} \;
+    - run: find . -mindepth 1 -maxdepth 1 -type f -name 'pip*' -exec rm {} \;
+      working-directory: ${{prefix}}/bin
 
     # make relocatable
-    cd {{prefix}}
-    for binfile in $shebangs $confdir/python-config.py; do
-      if ! test -f $binfile; then
-        binfile=${binfile/{{version.marketing}}/{{version.marketing}}$SUFFIX}
-      fi
-      if ! test -f $binfile; then
-        continue
-      fi
-      binfile=$(readlink -f $binfile)
-      sed -i.bak -e 's|#!{{ prefix }}/bin/|#!/usr/bin/env |g' $binfile
-      rm $binfile.bak
-    done
-
-    sed -i.bak -e 's|{{ prefix }}|\\$(shell tea --prefix)/python.org/v{{version.major}}|g' $confdir/Makefile
-    rm $confdir/Makefile.bak
+    - run: |
+        for binfile in $shebangs $confdir/python-config.py; do
+          if ! test -f $binfile; then
+            binfile=${binfile/{{version.marketing}}/{{version.marketing}}$SUFFIX}
+          fi
+          if ! test -f $binfile; then
+            continue
+          fi
+          binfile=$(readlink -f $binfile)
+          sed -i.bak -e 's|#!{{ prefix }}/bin/|#!/usr/bin/env |g' $binfile
+          rm $binfile.bak
+        done
+
+        sed -i.bak -e 's|{{ prefix }}|\\$(shell tea --prefix)/python.org/v{{version.major}}|g' $confdir/Makefile
+        rm $confdir/Makefile.bak
+      working-directory: ${{prefix}}
+
+    - run:
+        cat $PROP >> _sysconfigdata__*.py
+      working-directory:
+        ${{prefix}}/lib/python{{version.marketing}}
+      prop: |
+        import os
+        import re
+
+        tea_prefix = os.path.normpath(os.path.join(os.path.dirname(__file__), '../../../../'))
+        for key in build_time_vars:
+          if isinstance(build_time_vars[key], str):
+            build_time_vars[key] = re.sub(r'(\s|^)/opt/', r'\g<1>{}/'.format(tea_prefix), build_time_vars[key])
```

Comment:
nice

Reply:
yeah I think this is a pretty nice way to implement your requested feature